Makefile

```diff
@@ -72,7 +76,17 @@ drush:
 	$(call message,Executing \"drush -r /var/www/html/web $(COMMAND_ARGS) --yes\")
 	$(call docker-www-data, php drush -r /var/www/html/web $(COMMAND_ARGS) --yes)
 
-prepare: | up
+configuring_env:
+    # If OS Linux - change PHP_TAG in environment.
+	@if [ $(OS_NAME) = "Linux" ]; then \
+		echo "${cyan}${bold}$(PROJECT_NAME): Defined OS LINUX. Change PHP_TAG for docker image in environment${reset}" \
```

Comment:
We have a defined function for messages like this, see `$(call message,...)` examples in this file.